src/v2/view-builder/views/shared/ChallengePushView.js

```diff
@@ -28,7 +28,7 @@ const Body = BaseFormWithPolling.extend(Object.assign(
       // 'hasSavingState' would be true by default.
       // Setting it to false when auth key is okta_verify or custom_app with autochallenge schema
       // So that 'o-form-saving' css class is not added while polling and checkbox remains enabled.
-      if (this.isOVWithAutoChallenge() || this.isCustomAppWithAutoChallenge()) {
```

Comment:
I think the API should **not** return `autoChallenge` as part of the remediation form, since it sounds like it is not supported yet. That way we can change this file to hide the checkbox when that is the case